src/pages/index.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <HeroSection />
+</template>
+
+<script>
+import { CLEAR_FILTERS } from '~/store-modules/mutation-types'
+
+const HomePage = {
+  name: 'home-page',
+  fetch() {
+    this.$store.commit(CLEAR_FILTERS)
```

Comment:
Instead of using `fetch` here, `beforeMount` would suffice too. Mostly because there is no API logic involved and `fetchOnServer` is disabled anyway.